### DISCUSSION TITLE
Is it possible to use ASReview off line?

### DISCUSSION THREAD

**Discussion by JonnaLind** [asreview/asreview#1550] [Q&A]
Hi there, 

Is it possible to use ASReview without access to the internet, thus off line? 

Thanks in advance, 
Jonna Lind

**Comment by J535D165** [comment]
ASReview software is designed with privacy and transparency as core values. The software doesn't rely on external servers hosted by the ASReview team and should work perfectly without an internet connection. 

The following functionality doesn't work without an internet connection (non-exclusive list):
- Benchmark/test datasets in the Simulate or Explore modi
- Links to help files or donate buttons
- some models that rely on pre-trained, externally hosted models, most likely more advanced models.
